agent/ecscni/plugin_unsupported.go

```diff
@@ -23,6 +23,11 @@ import (
 	"github.com/containernetworking/cni/pkg/types/current"
 )
 
+const (
+	// vpcCNIPluginPath is the path of the cni plugin's log file.
+	vpcCNIPluginPath = "/log/vpc-branch-eni.log"
```

Comment:
is this needed?

Reply:
Yes. The method where this constant is used is in a common file (plugin.go)